plugins/modules/wafv2_ip_set.py

```diff
@@ -275,8 +281,9 @@ def main():
         description=dict(type='str'),
         ip_address_version=dict(type='str', choices=['IPV4', 'IPV6']),
         addresses=dict(type='list', elements='str'),
-        tags=dict(type='dict'),
-        purge_addresses=dict(type='bool', default=True)
+        tags=dict(type='dict', aliases=['resource_tags']),
+        purge_tags=dict(type='bool', default=True),
```

Comment:
```suggestion
        purge_tags=dict(type='bool', default=False),
```

in the first release